recipe/meta.yaml

```diff
@@ -35,6 +36,11 @@ requirements:
     # - geopandas >=0.3.0
 
 test:
+  requires:
+    - pip
+    - pytest
```

Comment:
I don't think it is good idea to run tests for plotnine. `scikit-misc` which is required for the tests is not on conda-forge yet. I have just moved it meson-build and hopefully that will make it easy to put it on conda-forge soon.

Reply:
Alright, I disabled pytest for now :)